src/components/Slider/index.tsx

```diff
@@ -76,36 +82,41 @@ const StyledBorderWrapper = styled(Box)`
   }
 `
 
-const Slider = ({ children, ...props }) => {
-  const scrollRef = useRef(null)
-  let intervalLeft
-  let intervalRight
+export const Item: FunctionComponent<{ as?: string | ElementType<unknown> }> =
+  ({ as, ...props }) => (
+    <StyledBorderWrapper as={as} {...props} draggable="true" />
+  )
+
+Item.propTypes = {
+  as: PropTypes.string,
```

Comment:
Seen with you, to be replaced by `ElementType`

Reply:
Seen with you, we keep as is until we remove the propTypes 😄